beginner_setup_guide.md

```diff
@@ -0,0 +1,164 @@
+
+### 1.3 Install arduino-esp8266fs-plugin
+
+Download the latest version of the ```arduino-esp8266fs-plugin``` from [here](https://github.com/esp8266/arduino-esp8266fs-plugin/releases/)
+
+In the Arduino IDE navigate to ```File -> Preferences``` and check for your ```Sketchbook location``` path. Navigate to that path in your file browser. If needed create the folder ```tools``` and extract the ```esp8266fs.jar``` into that folder.
+
+Restart the Arduino IDE after that.
+
+### 1.4 Download the websocket-gbp-emulator github code and additional data
+
+Download the full code from [here](https://github.com/HerrZatacke/websocket-gbp-emulator/archive/master.zip) as a ZIP file and extract it to your home directory. In your file browser navigate into the extracted folder and then into the ```websocket-gbp-emulator``` subfolder. Here create a copy of the ```config.h.sample.txt```. Rename the new file to ```config.h``` 
+
+It is recommended to download the additional data for the GB-Printer-Web interface. To automatically download the additional data do:
+
+#### Windows
+Go to your Start Menu, type ```powershell``` and navigate to the directory you just extracted (e.g. cd ```YourUsername\Downloads\websocket-gbp-emulator\```)
+Run the download script with ```.\update_w.ps1```
+
+#### MacOS / Linux
+Open your Terminal and navigate to the files you just extracted (e.g. ```cd ~/Downloads/websocket-gbp-emulator/```)
+Type ```chmod +x update_w.sh```
+Type ```./update_w.sh```
+
+### 1.5 Upload the code and data
+
+Connect your ESP8266 Board via USB to your PC. Navigate to ```File -> Open ...``` navigate to the folder you unzipped in the previous step. Open the ```websocket-gbp-emulator.ino``` 
+
+To confirm that the code was downloaded correctly navigate to ```Sketch -> Verify/Compile```. If everything works go to ```Sketch -> Upload``` and wait until the upload is finished.  
+
+Now nagivate to ```Tools -> ESP8266 Sketch Data Upload``` and wait until its finished.
+
+### 1.6  Test the board
+
+Unplug your ESP8266 Board from your PC and replug it. Open your WiFi Minitor and wait for it to finish scanning for available networks. You should see a wifi network named ```gameboyprinter``` with the password ```gameboyprinter```. 
+
+After you connected to the WiFi open your browser and type in the URL ```http://gameboyprinter.local```. The Gameboy Printer Web Emulator should load. If it doesn't please try to connect to ```https://192.168.4.1``` instead. 
+
+## Step 2: Preparing the Hardware
+
+Now that you've got your ESP8266 running with the websocket-gbp-emulator it's time to connect your Gameboy to it to actually copy your images. There are non-destructive ways ([Example](https://github.com/Palmr/gb-link-cable)) but for ease of access we will have to cut up a link cable. You can either obtain those cheap from ebay or find reproductions on websites like AliExpress. 
+
+### 2.1 Link Cable connection without a multimeter
+
```

Comment:
Maybe add a note that using a Multtimeter will be much easier? (especially because of the "corss-link" i mentionied below